core/tests/test_utils.py

```diff
@@ -220,7 +272,13 @@ def get_expected_logout_url(self, slug):
         return current_user_services.create_logout_url(slug)
 
     def _parse_json_response(self, json_response, expect_errors=False):
-        """Convert a JSON server response to an object (such as a dict)."""
+        """Convert a JSON server response to an object (such as a dict).
+
+        Args:
+            json_response: dict (json). Contains json response from api call.
```

Comment:
json_response: dict. The dict representation of the JSON response from the API call.